test/post.test.js

```diff
@@ -38,7 +39,12 @@ test.serial('Post run with github token', async t => {
   const debug = false;
   const tagName = `v${version}`;
   const options = {branch, debug, githubUrl, githubToken, githubApiPathPrefix};
-  const pkg = {version, repository: {url: `git+https://othertesturl.com/${owner}/${repo}.git`}};
+  const release = {};
+  const npm = {
```

Comment:
It's a small detail, but for readability and consistency, could you format the objects with few properties on one line? In other places as well?